DataDog/datadog-agent
# [ASCII-2240]  Avoid using the global tagger inside the logs agent

The logs agent internal packages use the tagger. 

Prior to this change it was relaying in the global tagger. With this changes, we are declaring the tagger a dependency of the Logs Agent and no longer using the global tagger. 

### QA

Unit test updated to ensure functionality remains the same. I might need some guidance from @DataDog/agent-metrics-logs to ensure this change requires manual QA or not.
